app/scripts/controllers/membership.js

```diff
@@ -364,5 +363,25 @@ angular
                 }
               });
             });
+
+
+          DataService
+            .list('serviceaccounts', requestContext)
```

Comment:
Yeah thats helpful.  I have the ability to enter an unlisted, but should def request & update whenever the namespace changes.